src/Components/List.jsx

```diff
@@ -0,0 +1,35 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { TodoItem } from './TodoItem';
+
+function List({ list, setTodoList, listName }) {
+  return (
+    <div
+      css={css({
+        height: '220px',
+        width: '100%',
+        padding: '18px 18px 0 18px',
+        borderBottom: listName == 'TO DO' ? 'solid 1px lightgray' : 'none',
+      })}
+    >
+      <div css={css({ fontSize: '18px', marginBottom: '15px' })}>
+        {listName} ({list.length})
+      </div>
+      <div
+        css={css({
+          width: '100%',
+          height: '156px',
+          overflowY: 'scroll',
+          display: 'flex',
+          flexDirection: 'column',
+          gap: '10px',
+          paddingRight: '20px',
+        })}
+      >
+        <TodoItem todoList={list} setTodoList={setTodoList} />
```

Comment:
네이밍이 어색한 부분이 있네요. TodoItem인데 list를 전달해주고 있어서 그런 것 같습니다. 현재 TodoItem의 역할이 단순 map만 돌고 있는데. 차라리 다음과 같이 하는 게 좋겠습니다.

```suggestion
  todoList.map((todo, index) => (
        <TodoItem key={index} todo={todo} setTodoList={setTodoList} />
   )
```